README.md

```diff
@@ -127,18 +127,6 @@ implementations have led to
 It is for this reason that this spec mandates that any random numbers used come from a
 "cryptographically secure" source, thereby (hopefully) avoiding such issues.
 
-### Why does the standard library API treat `v4` UUIDs as a default?
```

Comment:
@ctavan I think perhaps we should add two items to the FAQ:

1. something that points out the fact that v4 has the maximum amount of entropy that can exist, and still be called a UUID (pointing out that it sets the `reserved` and `version` fields must be set, to be a UUID, and version 4 fills the rest of the ID with randomness).
2. in a prior art section, let's point out the Java, Go, and C++ examples you bring up [here](https://github.com/tc39/proposal-uuid/issues/3#issuecomment-544190284)? I think this is very compelling.

Reply:
@bcoe I have added two paragraphs LMKWYT.